test/com/m3/octoparts/util/OctoMetricsImplSpec.scala

```diff
@@ -0,0 +1,44 @@
+package com.m3.octoparts.util
+
+import com.codahale.metrics.JvmAttributeGaugeSet
+import com.codahale.metrics.jvm.{ GarbageCollectorMetricSet, MemoryUsageGaugeSet, ThreadStatesGaugeSet }
+import com.m3.octoparts.support.PlayAppSupport
+import org.scalatest.{ BeforeAndAfterEach, Matchers, FunSpec }
+import scala.collection.JavaConverters._
+
+class OctoMetricsImplSpec extends FunSpec with Matchers with PlayAppSupport with BeforeAndAfterEach {
+
+  override protected def afterEach(): Unit = {
+    subject.onStop()
+  }
+
+  lazy val subject = new OctoMetricsImpl(appComponents.applicationLifecycle, appComponents.configuration)
+
+  describe("#onStart") {
+
+    it("should not throw") {
+      subject.onStart()
+    }
+
+    it("should work with consecutive calls") {
+      subject.onStart()
+      subject.onStart()
+    }
+
+    it("should work if metrics were already registered") {
```

Comment:
The original intent was to make it obvious within the test that *these* are the metrics we want to ensure are registered before calling `.onStart`, and to make that expectation explicit.

However, thinking about it again, it's just as easy to make it explicit by refactoring out the check into a separate method, call that method after calling `setupJvmMetrics`, and then call it again after calling `.onStart`. Done as part of 7a1e4fc